src/content/docs/en/welcome.mdx

```diff
@@ -47,10 +47,10 @@ After you're migrated, you'll get the latest and greatest Ultramarine features a
 
 ### Why does this distro exist? What difference does it have other than just Fedora with RPMFusion?
 
-Ultramarine Linux was created out of frustration with the legal limits of Fedora. As Fedora (and Red Hat) is an American entity, there are legal restrictions on what software can be
+Ultramarine Linux was created out of frustration with the legal challenges of Fedora. As Fedora's parent company Red Hat is an American entity, there are legal restrictions on what software can be
 packaged in the distribution due to the US patent system.
 
-Ultramarine, on the other hand, is based in Thailand and is not subject to any of the copyright restrictions on what software can be packaged.
+Ultramarine, on the other hand, is based in Thailand and is not subject to any of the copyright restrictions on what software can be packaged, preferring to give users the choice of what software to use based on the laws of their jurisdiction.

```

Comment:
preferring isn't quite right here, try "allowing users to chose software that works for them." 

aligns more with user choice and less with legal